### DISCUSSION TITLE
Postfix Admin version 3.3.11/3.3.13

### DISCUSSION THREAD

**Discussion by UTAVATU** [postfixadmin/postfixadmin#788] [General]
Dear Friends

What is the newest version of Postfix Admin 3.3.11 or 3.3.13? I have installed 3.3.13, but in the footer it still shows 3.3.11.

Sincerely

UTAVATU

**Comment by DavidGoodwin** [comment]
known problem; sorry ... someone forgot to update the version code in a release...
